content/applications/sales/crm/track_leads/unattended_leads_report.rst

```diff
@@ -0,0 +1,180 @@
+
+Click in the rule's second field, then select :guilabel:`is set` from the drop-down menu. Selecting
+this operator excludes any leads not assigned to a specific salesperson.
+
+Add a Sales team
+----------------
+
+Click :guilabel:`New Rule`. Click the first field for the new rule, and type `Sales Team` in the
+:guilabel:`Search...` bar, or scroll to search through the list to locate it.
+
+Click in the rule's second field, select :guilabel:`is in` from the drop-down menu. Selecting this
+operator limits results to the sales teams selected in the next field.
+
+In the third field, select the desired sales team from the drop-down menu. Multiple teams can be
+selected in this field.
+
+.. note::
+   This filter is optional. To view results for the entire company, do **not** add this filter.
+
+.. figure:: unattended_leads_report/configured-custom-rules.png
+   :align: center
+   :alt: An example of the Custom Filter pop-up window with all of the rules configured.
+
+   An example of the **Add Custom Filter** pop-up window with all of the rules configured.
+
+View results
+------------
+
+At the top of the :guilabel:`Add Custom Filter` form, there is an option to match :guilabel:`any` or
+:guilabel:`all` of the rules. In order to properly run the report, only records that match **all**
+of the following filters should be included. Before adding the filters, make sure :guilabel:`all` is
+selected in this field.
+
+.. image:: unattended_leads_report/all-custom-filter.png
+   :align: center
+   :alt: Example of overdue activities notification in the chatter of a lead.
+
+After the filters are configured, click :guilabel:`Add`. The resulting report displays all leads
+assigned to a salesperson where an activity is past due, or is due on the current date. The default
+display is a bar graph, where the leads are grouped by *stage*.
+
+To group the results by salesperson, click the down arrow to the right of the search bar. Under the
+:guilabel:`Group By` heading, select :guilabel:`Salesperson`.
+
+.. note::
+   The option to group by :guilabel:`Sales Team` is also available under the :guilabel:`Group By`
+   heading.
+
+To change to a *list* view, click the :guilabel:`≣ (bars)` icon in the top-right corner of the
+screen.
+
+.. tip::
+   Clicking the :guilabel:`(sliders) icon` opens a drop-down menu of additional columns that can be
+   added to the report.
+
+   Some options that are beneficial for this report include:
+
+   - :guilabel:`Activities`: the summary of the latest activity for this lead.
+   - :guilabel:`Expected Closing`: the estimated date on which the lead will be won.
+   - :guilabel:`Probability`: estimated success rate based on the stage.
+
+  .. image:: unattended_leads_report/additional-options.png
+     :align: center
+     :alt: Custom filter pop-up with emphasis on the options for activities and due date.
+
+.. _track_leads/unattended_leads_report/activity tracking:
+
+Activity tracking
```

Comment:
Not sure this section is necessary since the [Activities](https://www.odoo.com/documentation/17.0/applications/productivity/discuss/overview/plan_activities.html) doc already exists and was pointed to much earlier in the article. 

Replacement ideas:
- a **seealso** admonition pointing to [Activities](https://www.odoo.com/documentation/17.0/applications/productivity/discuss/overview/plan_activities.html) doc (again, as a redundancy)
- best practices for sales team success if they want to implement activities tracking for their team.
- just leave blank and just remove what's there (can always add more later if we want)

Reply:
I think just the see also is best. At some point, it will probably be worth it to do one "Activities in the CRM" doc, which I know we discussed in the past, that can be linked here. Currently, the activities doc is about the function of the feature, but if we want to go into how to utilize them within the CRM best specifically, I think it would be best to keep that in one separate doc, instead of adding that on as a footnote. Thoughts?

Reply:
> Thoughts?
Agree! I'll make a task.